REGION_FORK_RECOMMENDATIONS.md

```diff
@@ -0,0 +1,193 @@
+# Region Specific Fork Recommendations
+
+This document describes recommendations for keeping forks of [covidgreen/covid-green-backend-api](https://github.com/covidgreen/covid-green-backend-api) clean while also allowing for region specific changes. For example, the New York State fork [project-vagabond/covid-green-backend-api](https://github.com/covidgreen/covid-green-backend-api) contains CICD workflows specific to NYS which should not be contributed back to Covid Green. These recommendations adhere to the project's [Contributing Guidelines](CONTRIBUTING.md).
+
+## Goals and When to Use
+
+These recommendations exist for the situations when a Public Health Authority is basing their Covid Tracing system on Covid Green but potentially making region specific changes. There are 4 overall goals to these recommendations
+  
+1. All changes, unless explicitly region specific, will be contributed back to [covidgreen/covid-green-backend-api](https://github.com/covidgreen/covid-green-backend-api).
+1. Region specific forks will stay up-to-date with [covidgreen/covid-green-backend-api](https://github.com/covidgreen/covid-green-backend-api).
+1. Region specific changes will never be contributed back to [covidgreen/covid-green-backend-api](https://github.com/covidgreen/covid-green-backend-api).
+1. Region specific changes will be limited in scope and never cause a merge conflict
+
+These recommendations exist because it is reasonable to assume that individual Public Health Authorities may want to customize specific pieces of the Covid Green codebase, but wish to do so in a way that will not cause explosive merge conflicts. These recommendations work best when the set of region specific changes are limited in scope, and unless explicitly region specific, all changes are contributed back to [covidgreen/covid-green-backend-api](https://github.com/covidgreen/covid-green-backend-api).
+
+As an example, [New York State's fork](https://github.com/project-vagabond/covid-green-backend-api) contains NYS specific workflow differences which cannot be contributed back to Covid Green.
+
+## Branches
+
+The following items are assumed to be true for these recommendations to apply.
+
+1. `current` is the main development branch within Covid Green. 
+1. Covid Green's `current` branch may be merged into any fork's `current` branch without causing merge conflicts.
+1. `current` will never include region specific content
```

Comment:
Just noting, we'll obviously need to work on replacing the Ireland specific content that's currently in `current`...